README.md

```diff
@@ -65,6 +65,27 @@ build_index(embeddings="embeddings", index_path="my_index_folder/knn.index",
             current_memory_available="4G")
 ```
 
+## Memory-mapped indices
+
+Faiss makes it possible to use memory-mapped indices. This is useful when you don't need a fast search time (>50ms)
+and still want to reduce the memory footprint to the minimum.
+
+We provide the should_be_memory_mappable boolean in build_index function to generate memory-mapped indices only.
+Note: Only IVF indices can be memory-mapped in faiss, so the output index will be a IVF index.
+
+To load an index in memory mapping mode, use the following code:
+```python
+import faiss
+faiss.read_index("my_index_folder/knn.index", faiss.IO_FLAG_MMAP | faiss.IO_FLAG_READ_ONLY)
+```
+
+You can have a look to the [examples](examples/memory_mapped_autofaiss.py) to see how to use it.
+
+Technical note: You can create a direct map on IVF indices with .make_direct_map() in order to speed up
```

Comment:
probably mention here the option to do that that martin added in autofaiss

Reply:
Yep!